.prettierignore

```diff
@@ -13,7 +13,6 @@
 !*.ts
 !*.tsx
 !*.yml
-
```

Comment:
It’d be great to remove this file from the PR diff

Reply:
You can try copy-pasting these two files from the `main` branch to make them disappear from the diff.